acme/acme/challenges.py

```diff
@@ -216,6 +218,90 @@ def response_and_validation(self, account_key, *args, **kwargs):
 
 
 @ChallengeResponse.register
+class DNS01Response(KeyAuthorizationChallengeResponse):
+    """ACME "dns-01" challenge response."""
+    typ = "dns-01"
+
+    def txt_records_for_name(self, name):
```

Comment:
`self` is not used :( if this passes tests then that means that `pylint` config has been messed up

@pde I still feel like a robot :(